keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/ImageProvider.kt

```diff
@@ -35,7 +37,8 @@ sealed class ImageProvider
 @JsonDeserialize(using = JsonDeserializer.None::class)
 data class ArtifactImageProvider(
   val deliveryArtifact: DeliveryArtifact,
-  val artifactStatuses: List<ArtifactStatus> = enumValues<ArtifactStatus>().toList()
+  @JsonInclude(NON_EMPTY)
+  val artifactStatuses: List<ArtifactStatus> = emptyList() // treated as "all statuses" by ImageResolver
```

Comment:
You changed that behavior?

Reply:
Yeah, the `ImageResolver` change in this PR accompanies this, it treats an empty `artifactStatuses` as `enumValues<ArtifactStatus>().toList()` which helps with omitting listing all statuses when newly exporting a cluster.